tests/ga/test_extension.py

```diff
@@ -27,6 +27,12 @@
 from azurelinuxagent.ga.exthandlers import *
 from azurelinuxagent.common.protocol.wire import WireProtocol, InVMArtifactsProfile
 
+SLEEP = time.sleep
```

Comment:
add comments

Reply:
Ah sorry, I've been adding this to all my PRs ever since @larohra discovered this magical mock :) forgot to add a comment to this instance.

Reply:
Minor - Can you rename the var to MOCK_SLEEP? Only SLEEP felt like a boolean to me.